api/environments/dynamodb/dynamodb_wrapper.py

```diff
@@ -110,3 +110,9 @@ def write_environments(self, environments: Iterable[Environment]):
         with self._table.batch_writer() as writer:
             for environment in environments:
                 writer.put_item(Item=build_environment_document(environment))
+
+    def get_item(self, api_key: str) -> dict:
+        try:
+            return self._table.get_item(Key={"api_key": api_key})["Item"]
+        except KeyError as e:
```

Comment:
Is this correct? I have a feeling that we changed something recently to inspect whether the `"Item"` attribute was `None`?

Reply:
Yeah, that was around query Item returning empty list